README.md

```diff
@@ -8,7 +8,9 @@ the source for the generation of the Flattened Image Tree Specification using
 Sphinx and LaTeX.
 
 ## Mailing list: flat-image-tree@lists.osfw.foundation ##
-   * Use this mailing list for submitting patches, questions and general discussion
+   * Use this mailing list for submitting questions and general discussion
+   * Patches should be submitted via a pull-request at
+     https://github.com/open-source-firmware/flat-image-tree/pulls
    * Sign up to the mailing list at https://lists.osfw.foundation/g/flat-image-tree
```

Comment:
suggestion (non-blocking): reorder

It's a bit odd to say to use the mailing list for things A, but patches another way, but hey this is how to sign up to the mailing list.

I would suggest to reorder the list such that
>  Patches should be submitted via a pull-request at https://github.com/open-source-firmware/flat-image-tree/pulls

be either first or last in the list.

Reply:
done, I think, but please check that I understood you correctly.